shelley/chain-and-ledger/shelley-spec-ledger-test/src/Test/Shelley/Spec/Ledger/Generator/Constants.hs

```diff
@@ -129,7 +129,7 @@ defaultConstants =
       frequencyPotentiallyManyWithdrawals = 5,
       minSlotTrace = 1000,
       maxSlotTrace = 5000,
-      frequencyLowMaxEpoch = 6,
+      frequencyLowMaxEpoch = 200,
```

Comment:
now that we check for Epoch wellformedness in the Pool properties, we need to generate sufficiently large Max epochs!